UILibraries/ImageSlider/ImageSlider/Classes/Core/ImageSliderCollectionViewCell/ImageSliderCollectionViewCell.swift

```diff
@@ -15,7 +15,7 @@ protocol ImageSliderCollectionViewCellInterface: class {
 }
 
 final class ImageSliderCollectionViewCell: UICollectionViewCell {
-    lazy var contentImageView: PinchableImageView = {
+    private (set) lazy var contentImageView: PinchableImageView = {
```

Comment:
you can remove the unnecessary space between `private` and `(set)` -> `private(set)`.

Reply:
There is no strict rule about that. It's about the code styling and looks better with a space to me. 😊